### ISSUE TITLE
Add changelog and/ or release notes

### ISSUE THREAD

**Issue by cwerner** [webdataset/webdataset#119]
Hi :wave:

Happy user of `webdataset` here. Thanks for your great library and hard work! 

I found the project some months back and used it for a students DL project to simplify the workflow and dataset handling and it has been great so far! So no large-scale deployment but a desktop DL setting.

One suggestion though: It would really help if there was a Release Notes and/ or ChangeLog file to judge if we should jump versions and to learn about new features. The documentation is good, but a quick peak on what's changed would be great!

At the moment I have the project pinned to v0.1.62 since there apparently was some API change afterwards and my code breaks/ would need to be adapted to that (which I'll do soon but had to postpone for the moment).

Thanks 👍

**Comment by tmbdev** [comment]
Yes, I agree that better release management, change logs, and version management would be desirable and I'm going to try to improve that.

Pinning at a particular version is probably the best thing to do for now. The other thing you can do is just install from PyPI, since I make packaged releases only when things are fairly stable.
